sqlglot/executor/env.py

```diff
@@ -16,6 +16,41 @@ def __lt__(self, other):
         return other.obj < self.obj
 
 
+def filter_nulls(func):
+    def _func(values):
+        return func(v for v in values if v is not None)
+
+    return _func
+
+
+def str_position(substr, this, position=None):
+    if substr is None or this is None:
```

Comment:
is there a way we can have the helper do this as well? maybe like a decorator to return None if any arg is None?